### PR TITLE
DCKA-121 add readme to wallet core

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

File: packages/core/readme.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# React native SDK for Dock app
+
+Using [polkadot-js](https://polkadot.js.org/) libraries in react native is a challange, due to a lack of webassembly support.
```
Comment (mike-parkhill) [inline packages/core/readme.md:3]: nitpick: should use the accepted spelling and capitalization:  `WebAssembly`

File: packages/core/readme.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+The Dock Wallet SDK handles all the Polkadot web assembly in a webview, sending messages to the react native thread trough a JSON RPC layer.
```
Comment (mike-parkhill) [inline packages/core/readme.md:5]: typo: `trough` -> `through`

should also update `WebAssembly`

File: packages/core/readme.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    </Box>
+  );
+};
+
+const App = () => {
+  return (
+    <NativeBaseProvider>
+      <WalletSDKProvider>
+        <Box p={8}>
+          <Text>SDK Demo</Text>
```
Comment (mike-parkhill) [inline packages/core/readme.md:49]: should probably be `Dock Wallet SDK Demo`. 

SDK is fairly ambiguous.

File: packages/transactions/readme.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+
+# Transactions Manager
+
+Transactions manager for the dock wallet sdk
```
Comment (mike-parkhill) [inline packages/transactions/readme.md:4]: Should maybe define what transactions actually are. Probably would require a reference to the Dock token here somewhere.

File: packages/dids/readme.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+
+# DID Manager
+
+DID manager for the dock wallet sdk
```
Comment (mike-parkhill) [inline packages/dids/readme.md:4]: either define DIDs or link to our API docs where we have definitions of DIDs, etc. (or both)

**Review by mike-parkhill — CHANGES_REQUESTED** [review]
General consistency in naming. Always use proper names for projects/libraries
i.e. WebAssembly, Dock, React Native, etc.

File: packages/credentials/readme.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
+
+# Credentials manager
+
+Credential manager for the dock wallet sdk
```
[the remark this answers is not in the document]
Comment (maycon-mello) [reply packages/credentials/readme.md]: Fixed

File: packages/dids/readme.md (hunk 5 above)
Comment (maycon-mello) [reply packages/dids/readme.md]: Fixed

File: packages/transactions/readme.md (hunk 4 above)
Comment (maycon-mello) [reply packages/transactions/readme.md]: Fixed

File: packages/transactions/readme.md [hunk 7]
```diff
@@ -1,9 +1,11 @@
 
 # Transactions Manager
 
-Transactions manager for the dock wallet sdk
+Transactions are required to send and receive (DOCK Tokens)[[https://www.dock.io/token]. This module handles the transaction history and provie methods to send DOCK tokens to any address.
```
Comment (mike-parkhill) [inline packages/transactions/readme.md:4]: typo: `provie` -> `provides`

File: packages/dids/readme.md [hunk 8]
```diff
@@ -1,9 +1,9 @@
 
 # DID Manager
 
-DID manager for the dock wallet sdk
+DID stands for Decentralized IDentifiers. DIDs are meant to be globally unique identifiers that allow their owner to prove cryptographic control over them. A DID identifies any subject (e.g., a person, organization, thing, data model, abstract entity, etc.) that the controller of the DID decides that it identifies.
```
Comment (mike-parkhill) [inline packages/dids/readme.md:4]: should we call out which DID methods we support here somewhere?

Comment (maycon-mello) [reply packages/dids/readme.md:4]: I think it supports just DID key, right @osinakayah?

Comment (mike-parkhill) [reply packages/dids/readme.md:4]: Per the thread in Slack let's add `did:dock` as well.

**Review by mike-parkhill — COMMENTED** [review]
I think just add the supported DIDs list in the dids package and I think we're good.

File: packages/dids/readme.md [hunk 9]
```diff
@@ -5,6 +5,8 @@ DID stands for Decentralized IDentifiers. DIDs are meant to be globally unique i
 
 On React Native it's required to set up the WalletSDKProvider, described on [@docknetwork/wallet-sdk-react-native](https://www.npmjs.com/package/@docknetwork/wallet-sdk-react-native)
 
+Supports did:key
```
Comment (mike-parkhill) [inline packages/dids/readme.md:8]: and did:dock

Comment (mike-parkhill) [reply packages/dids/readme.md:8]: or are you planning on updating this when we finish the current work?

**Pull request merged**